mesonbuild/scripts/uninstall.py

```diff
@@ -24,7 +24,10 @@ def do_uninstall(log):
             continue
         fname = line.strip()
         try:
-            os.unlink(fname)
+            if os.path.isdir(fname):
```

Comment:
This returns true if fname is a symlink that points to a dir. It should probably have `follow_symlinks=False`.